packages/test-runner-commands/test/emulate-hover/emulateHoverPlugin.test.ts

```diff
@@ -0,0 +1,122 @@
+import path from 'path';
+import { platform } from 'os';
+import selenium from 'selenium-standalone';
+import { TestRunnerPlugin } from '@web/test-runner-core';
+import { runTests } from '@web/test-runner-core/test-helpers';
+import { chromeLauncher } from '@web/test-runner-chrome';
+import { webdriverLauncher } from '@web/test-runner-webdriver';
+import { playwrightLauncher } from '@web/test-runner-playwright';
+import { visualRegressionPlugin } from '@web/test-runner-visual-regression/plugin';
+
+import { emulateHoverPlugin } from '../../src/emulateHoverPlugin';
+
+import { startSeleniumServer } from '../selenium-server.js';
```

Comment:
Let's convert the file to `selenium-server.ts` to mitigate "Could not find a declaration file" error.

Reply:
Done.